docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more Autofill services to be installed on a Tizen device. Autofill manager is a module for managing the installed autofill services. Application developers can use autofill manager to get the current autofill service or change the autofill service.
+
+The main features of the Autofill Manager API include:
+
+- Installed autofill service list
```

Comment:
Please change to:
Installing autofill service list

Reply:
@imfine98, okay, thanks for the change
It would be good to change it to:
Retrieve installed autofill service list